requests-for-comments/commnunity-2024.md

```diff
@@ -0,0 +1,47 @@
+Up to now, much of our thinking for the community section of the website has centered on creating a Hacker News-like experience where content drives the community. I think this is the wrong direction. This issue outlines why and an alternative vision of the community.
+
+## Why not the current direction?
+
+Content, both our own and good content curated _by us_ can still form an important part of our community, but:
+
+1. There are already numerous places to share content about building startups / engineering topics generally. Why should people post it to ours and not another, more established channel?  
+
+2. The attraction of our content comes from our strong POV and curation. Outsourcing curation to the community undermines a strength we already have.
+
+3. If it were successful, it would create a significant moderation challenge that would suck up a bunch of time. Abuse and self-promotion will happen and we'll have to manage that.
+
+4. There's a significant risk our own content would be drowned out by community contributions and submissions to the point.
+
+5. It doesn't create any kind of stickiness for our product.
+
+## An alternative vision – HogBook
+
+I think the best way to create a thriving community on the website is to focus it around the product. This means:
+
+- Making it easy to request new features and vote on what the teams are working on
+- Making it easy to see what teams are working on and their progress
+- Creating a direct connection between all users and our engineers (not just the teams who pay for a dedicated Slack channel)
+- Creating a feedback loop where users request / vote on a feature then get notified when it's shipped
+- Creating a space where users can share feedback in public, rather than it being hidden in Zendesk / Slack channels.
```

Comment:
We do this already to some extent at a company level, but I do like this idea of taking it a step further to a team level. It will definitely foster creating direct connections with individuals on PostHog, and ultimately be a good user experience for our users.